src/main/java/com/github/tomakehurst/wiremock/standalone/CommandLineOptions.java

```diff
@@ -113,6 +113,7 @@
     private static final String HTTPS_CA_KEYSTORE = "ca-keystore";
     private static final String HTTPS_CA_KEYSTORE_PASSWORD = "ca-keystore-password";
     private static final String HTTPS_CA_KEYSTORE_TYPE = "ca-keystore-type";
+    private static final String ROOT_DIR_UNDER_CLASSPATH = "root-dir-under-classpath";
```

Comment:
Maybe `load-mappings-from-classpath` would be a better config key for this?

Reply:
That's fair - I guess this will be both `__files` and `mappings` - is `load-resources-from-classpath` OK to cover both?

Reply:
Yes, I guess "resources" is better since it's both. Let's go with that.